build.gradle

```diff
@@ -33,7 +33,7 @@ ext {
     gebVersion = '3.4'
     unirestVersion = "3.7.02"
     seleniumVersion = '3.141.59'
-    chromeDriverVersion = '86.0.4240.22'
+    chromeDriverVersion = '87.0.4280.20'
```

Comment:
The best approach is to use https://github.com/bonigarcia/webdrivermanager, otherwise you have to update it every time Chrome changes its version...

Reply:
We will include this in the next iteration

Reply:
thanks for this information